packages/cli/interactive.ts

```diff
@@ -304,7 +311,7 @@ const getTextRenderers = (config: TextPromptConfig) => {
 		],
 		submit: ({ value }: { value: Arg }) =>
 			renderSubmit(config, format(value ?? "")),
-		cancel: handleCancel,
+		cancel: activeRenderer,
```

Comment:
Clack will re-render the prompt on cancelled state. This allows us to keep rendering what users see before we terminate the process.

Before this change, we were terminating the process directly so it never has a chance to re-render. But it is now delayed to the `isCancel` check after the prompt resolves with a value.